processes/consumer/kafka.go

```diff
@@ -89,7 +90,9 @@ func StartConsumer(ctx context.Context, cfg config.Config, inMemDB *models.Datab
 	}
 
 	var wg sync.WaitGroup
-	for _, topic := range topics {
+	for num, topic := range topics {
+		// It is recommended to not try to establish a connection all at the same time, which may overwhelm the Kafka cluster.
+		time.Sleep(jitter.Jitter(100, 3000, num))
```

Comment:
I don't think we want to use `Jitter` here because it's an exponential algorithm and we probably want something that's more linear. For instance, when num is `4` the upper bound will be 1600ms but `5` and higher will produce `3000`, so then everything will essentially wait 3 seconds and then try to connect at the same time.

Reply:
Maybe `rand.Int63(100 * n)`.

Reply:
What do you suggest?

Reply:
`(200 * num) * time.Millisecond`?

Reply:
Although I suppose that is just the upper bound so it'll be somewhat random within that.

Reply:
>so then everything will essentially wait 3 seconds and then try to connect at the same time.
is not accurate.

Reply:
Yeah it's not true exponential, it has randomness

```
random_between(0, min(cap, base * 2 ** attempt))
```